src/components/select-field/__tests__/BaseSelectField.test.js

```diff
@@ -242,6 +242,20 @@ describe('components/select-field/BaseSelectField', () => {
                 expect(overlay.hasClass(OVERLAY_SCROLLABLE_CLASS)).toBe(result);
             },
         );
+
+        test('should apply preventOverflow modifier when isEscapedWithReference is true', () => {
+            const wrapper = shallowRenderSelectField({ isEscapedWithReference: true });
+
+            const props = wrapper.find('PopperComponent').props();
+            expect(props.modifiers.preventOverflow).toBeDefined();
```

Comment:
nit: Can we use `toEqual` in order to also check that `isEscapedWithReference` is passed correctly?